flink-runtime-web/web-dashboard/src/app/pages/job/overview/backpressure/job-overview-drawer-backpressure.component.html

```diff
@@ -63,9 +63,18 @@
   </thead>
   <tbody>
     <ng-template nz-virtual-scroll let-data>
-      <ng-container *ngIf="narrowLogData(data) as subtask">
+      <ng-container *ngIf="narrowType(data) as subtask">
         <tr>
-          <td>{{ subtask['subtask'] }}</td>
+          <td
+            [nzShowExpand]="subtask['other-concurrent-attempts']?.length > 0"
+            [nzExpand]="expandSet.has(subtask['subtask'])"
+            (nzExpandChange)="onExpandChange(subtask, $event)"
+          >
+            {{ subtask['subtask'] }}
+            <ng-container *ngIf="subtask['attempt-number'] !== undefined">
+              &nbsp;[attempt-{{ subtask['attempt-number'] + 1 }}]
```

Comment:
nit: I think it's better to use `padding` or `margin` instead of `&nbsp;`, but it's a problem :)